components/earnings/commons/earnings_base.lua

```diff
@@ -129,21 +128,21 @@ function Earnings.calculatePerYear(conditions, divisionFactor)
 	local totalEarningsByYear = {}
 	local earningsData = {}
 	local totalEarnings = 0
+	local prizePoolColumn = Earnings._getPrizePoolType(divisionFactor)
 
 	local offset = 0
 	local count = _MAX_QUERY_LIMIT
 	while count == _MAX_QUERY_LIMIT do
 		local lpdbQueryData = mw.ext.LiquipediaDB.lpdb('placement', {
 			conditions = conditions,
-			query = 'prizemoney, mode, date',
+			query = 'mode, date, ' .. prizePoolColumn,
 			limit = _MAX_QUERY_LIMIT,
 			offset = offset
 		})
 		for _, item in pairs(lpdbQueryData) do
-			local prizeMoney = tonumber(item.prizemoney) or 0
 			local year = string.sub(item.date, 1, 4)
-			prizeMoney = prizeMoney / divisionFactor(item['mode'])
-			earningsData[year] = (earningsData[year] or 0) + prizeMoney
+			local prizeMoney = tonumber(item[prizePoolColumn]) or 0
```

Comment:
There's a tonumber here (and a `or 0`), while line 117 doesn't have that. Feels like both either needs it or neither needs it?